index.html

```diff
@@ -8,21 +8,79 @@
     <title>&#128587;&#127995;&#8205;&#9792;&#65039;</title>
   </head>
   <body>
-    <header>
-      <h1>Hi, ich bin Magdalena.</h1>
+    <header id="header">
+      <nav>
+        <ul>
+          <li><a href="#header">Home</a></li>
+          <li><a href="#lorem-1">Lorem-1</a></li>
+          <li><a href="#lorem-2">Lorem-2</a></li>
+          <li><a href="#lorem-3">Lorem-3</a></li>
+          <li><a href="url">Contact</a></li>
+        </ul>
+      </nav>
+      <h1>Hi, ich bin <i>Name</i>.</h1>
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Rem nihil
+        totam natus ut libero distinctio odit est nemo eaque placeat?
+      </p>
+      <a href="#lorem-1"><button>Learn more</button></a>
```

Comment:
```suggestion
      <a href="#lorem-1">Learn more</a>
```
You don't need a button since the `a` tag is clickable.